src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePickerFlyout.iOS.cs

```diff
@@ -1,58 +1,79 @@
 ﻿#if XAMARIN_IOS
+
 using System;
 using System.Collections.Generic;
 using System.Text;
+using Uno.UI;
 using Uno.UI.Common;
+using Uno.UI.Helpers.WinUI;
+using Windows.UI.Xaml.Automation;
 using Windows.UI.Xaml.Controls.Primitives;
 using Windows.UI.Xaml.Data;
 
 namespace Windows.UI.Xaml.Controls
 {
-	//TODO: should inherit from PickerFlyoutBase
 	public partial class TimePickerFlyout : Flyout
 	{
-		private bool _hasAcceptButton = false;
+        private bool AlreadyCancelledOrSaved { get; set; }
 
 		public TimePickerFlyout()
 		{
 			InitializeContent();
 		}
 
-		private void InitializeContent()
+		private void onTap(object sender, Input.PointerRoutedEventArgs e) => e.Handled = true;
```

Comment:
I believe this OnTap that sets the event as handled would benefit from some comments explaining why it's there :)